README.md

```diff
@@ -121,6 +121,111 @@ connection
 
 Binding also allowed positional index (zero-based) references.  The parameter index is derived from the parameter discovery order when parsing the query.
 
+
+## Data Type Mapping
+
+This reference table shows the type mapping between [PostgreSQL][p] and Java data types:
+
+| PostgreSQL Type                                | Supported Data Type                                                                                                                           | 
+|:-----------------------------------------------|:----------------------------------------------------------------------------------------------------------------------------------------------|
+| [`bigint`][psql-bigint-ref]                    | [**`Long`**][java-long-ref], [`Boolean`][java-boolean-ref], [`Byte`][java-byte-ref], [`Short`][java-short-ref], [`Integer`][java-integer-ref] |
+| [`bit`][psql-bit-ref]                          | Not yet supported.|
+| [`bit varying`][psql-bit-ref]                  | Not yet supported.|
+| [`boolean or bool`][psql-boolean-ref]          | [`Boolean`][java-boolean-ref]|
+| [`box`][psql-box-ref]                          | Not yet supported.|
+| [`bytea`][psql-bytea-ref]                      | Not yet supported.|
+| [`character`][psql-character-ref]              | [`String`][java-string-ref]|
+| [`character varying`][psql-character-ref]      | [`String[]`][java-string-ref]|
```

Comment:
`VARCHAR` maps to `String`

Reply:
Changed to `String` instead of `String[]`